os/include/tinyara/mm/mm.h

```diff
@@ -245,6 +245,15 @@ typedef size_t mmsize_t;
 #define MMSIZE_MAX SIZE_MAX
 #endif
 
+#define REMOVE_NODE(a)					\
```

Comment:
1. Let's remove it to say what it does.
   `REMOVE_NODE_FROM_LIST` or `REMOVE_NODE_IN_LIST` or something else
2. It is not called from outside. Let's add this in the header of os/mm/mm_heap folder.